furusiya/model/entities/party/player.py

```diff
@@ -36,17 +36,37 @@ def __init__(self):
 
         print("You hold your wicked-looking {} at the ready!".format(weapon_name))
 
+    def move_or_attack(self, dx, dy):
+        # TODO: Should this be part of the Fighter component?
```

Comment:
Agreed that this could be part of the fighter component. Not entirely sure though. I think it's fine here for now.